src/coreclr/interpreter/compiler.cpp

```diff
@@ -4306,11 +4304,11 @@ void InterpCompiler::EmitLdind(InterpType interpType, CORINFO_CLASS_HANDLE clsHn
     m_pLastNewIns->SetDVar(m_pStackPointer[-1].var);
 }
 
-void InterpCompiler::EmitStind(InterpType interpType, CORINFO_CLASS_HANDLE clsHnd, int32_t offset, bool reverseSVarOrder)
+void InterpCompiler::EmitStind(InterpType interpType, CORINFO_CLASS_HANDLE clsHnd, int32_t offset, bool reverseSVarOrder, bool enableImplicitArgConversionRules)
 {
     int stackIndexValue;
     int stackIndexAddress;
-    if (reverseSVarOrder)
+    if (!reverseSVarOrder)
```

Comment:
Is this changing because it was wrong before? If so, how did we not notice?

Reply:
We didn't actually have a test case where it matters... it turns out that Roslyn won't do this often, and most of the IL tests don't either.